clock/README.md

```diff
@@ -0,0 +1,268 @@
+# Quartz
+A Go time testing library for writing deterministic unit tests
+
+_Note: Quartz is the name I'm targeting for the standalone open source project when we spin this
+out._
+
+## Why another time testing library?
+
+Writing good unit tests for components and functions that use the `time` package is difficult, and
+several open source libraries exist.  In building Quartz, we took some inspiration from
+
+ * [github.com/benbjohnson/clock](https://github.com/benbjohnson/clock)
+ * Tailscale's [tstest.Clock](https://github.com/coder/tailscale/blob/main/tstest/clock.go)
+ * [github.com/aspenmesh/tock](https://github.com/aspenmesh/tock)
+
+Quartz shares the high level design of a `Clock` interface that closely resembles the functions in
+the `time` standard library, and a "real" clock passes thru to the standard libary in production,
+while a mock clock gives precise control in testing.
+
+Our high level goal is to write unit tests that
+
+1. execute quickly
+2. don't flake
```

Comment:
suggestion: avoid colloquialism
```suggestion
2. execute reliably
```

Reply:
While arguably "jargon", I think flaky is more precise. It refers to tests that succeed or fail due to uncontrolled randomness.

A test might not be reliable for other reasons, such as not actually testing or asserting the thing is purports to.